app/views/newPaymentFlow.scala.html

```diff
@@ -1,106 +1,110 @@
 <header role="banner">
-  <a href="#"><img src="#" alt="The Guardian"></a>
+  <a class="glogo" href="#"><img src="#" alt="The Guardian"></a>
```

Comment:
In order for the code style to diverge from the rest of the project I would avoid building twirl views and use jsx components instead.

Reply:
I will refactor into JSX later, right now I want to get the rendering correct asap and I work faster in vanilla HTML